From f4356a52f12d3d8a7e43badc1b316bcbc4943eff Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Wed, 7 Dec 2011 13:40:55 +0800
Subject: [PATCH 0697/2463] ENGR00161867 MX6Q: change
 CONFIG_FORCE_MAX_ZONEORDER from 13 to 14

Fix the bug: can't allocate buffer if setting resolution to 1920*1080p

Bug detailed description:
use 1920,1080 output resolution
mxc_sdc_fb mxc_sdc_fb.0: Unable to allocate framebuffer memory

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 0f2c0e8..f7ffa37 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -299,7 +299,7 @@ CONFIG_IMX_HAVE_PLATFORM_IMX_MIPI_CSI2=y
 # CONFIG_ARCH_MXC91231 is not set
 # CONFIG_ARCH_MX5 is not set
 CONFIG_ARCH_MX6=y
-CONFIG_FORCE_MAX_ZONEORDER=13
+CONFIG_FORCE_MAX_ZONEORDER=14
 CONFIG_ARCH_MX6Q=y
 CONFIG_SOC_IMX6Q=y
 CONFIG_MACH_MX6Q_ARM2=y
-- 
1.7.7.4

